Packages/vcs/vcs/vcs2vtk.py

```diff
@@ -251,11 +251,18 @@ def genGridOnPoints(data1, gm, deep=True, grid=None, geo=None,
     if geo is None:
         bounds = pts.GetBounds()
         xm, xM, ym, yM = [bounds[0], bounds[1], bounds[2], bounds[3]]
-        # We don't use the zooming feature (gm.datawc) for geographic
-        # projections. We use wrapped coordinates for doing the projection
+        # We use zooming feature (gm.datawc) for linear and ploar projections.
+        # We use wrapped coordinates for doing the projection
```

Comment:
this sentence does not read well --> such that parameters such that central meridian are set correctly

@danlipsa could you please clean it up?